### PR TITLE
add an option to pass the binary from cli

### PR DESCRIPTION
**Pull request by thib3113**
passing binary from cli allow shell to autocomplete the path of the binary, not the read .

( that's a things that frustrated me the first time, I doesn't know the exact path to my script )

### CODE REVIEW

**Review by sbs2001 — APPROVED** [review]
LGTM, except relative path don't work ;)

File: scripts/install.sh [hunk 1]
```diff
@@ -7,6 +7,40 @@ SYSTEMD_PATH_FILE="/etc/systemd/system/cs-custom-bouncer.service"
...
+    key="${1}"
+    case ${key} in
+    -b|--binary)
+        if ! [ -f "${2}" ]; then
+            echo "${key} need a path"
+            usage
+            exit 1
+        fi
+        BINARY_PATH="$2"
+        shift # past argument
```
Comment (sbs2001) [inline scripts/install.sh:29]: Doing some validation of whether the path is absolute would be helpful. 
```suggestion
        shift # past argument
        if ! [[ $BINARY_PATH == /* ]]; then
            echo "binary path must be absolute"
            exit 1
        fi
```

Comment (thib3113) [reply scripts/install.sh:29]: I think changing doc ( https://github.com/crowdsecurity/cs-custom-bouncer/pull/10/files#diff-e076e43cb817ecc8b3c2ed18cd58baa50544ecc66293ed6e61129c43ca6a09ccR24 ) to `need an absolute path` can be a good idea too .

Comment (sbs2001) [reply scripts/install.sh:29]: Let's support relative path too: 
```suggestion
        shift # past argument
        BINARY_PATH=$(readlink -f $BINARY_PATH)
```
Just adding this one line does the trick :) 
Credit to @AlteredCoder

**Pull request merged**